about/view/client/about.css

```diff
@@ -0,0 +1,23 @@
+.section-padding {
+    padding-top: 3em;
+    padding-bottom: 3em;
+}
+
+#about-banner {
+    padding: 70px 0;
```

Comment:
I borrowed this from the landing page since it is the same until we get a movie to display, but I can try to make it work with em or % instead.